RevenueCatUI/Resources/en.lproj/Localizable.strings

```diff
@@ -6,3 +6,10 @@
 "Restore purchases" = "Restore purchases";
 "Terms" = "Terms";
 "Terms and conditions" = "Terms and conditions";
+
+"PackageType.annual" = "Annual";
+"PackageType.sixMonth" = "6-month";
+"PackageType.threeMonth" = "3-month";
+"PackageType.twoMonth" = "2-month";
```

Comment:
Should these be `6 month`, `3 month`, and `2 month`? 🤔

Reply:
Changed